src-docs/src/views/header/header.js

```diff
@@ -64,7 +64,12 @@ export default () => {
       },
     ];
 
-    return <EuiHeaderBreadcrumbs breadcrumbs={breadcrumbs} />;
+    return (
+      <EuiHeaderBreadcrumbs
+        aria-label="breadcrumbs example"
```

Comment:
Don't forget to use proper sentence casing even in aria-labels.

```suggestion
        aria-label="Header breadcrumbs example"
```